src/app/component/dashboard/dashboard.module.ts

```diff
@@ -102,6 +102,24 @@ import { InvoiceComponent } from './invoice/invoice.component';
 import { InvoicePrefixComponent } from './administrator/invoice-prefix/invoice-prefix.component';
 import { SmsBirthdayCongratulationComponent } from './parameters/sms-birthday-congratulation/sms-birthday-congratulation.component';
 import { MailBirthdayCongratulationComponent } from './parameters/mail-birthday-congratulation/mail-birthday-congratulation.component';
+import { HelpCenterComponent } from './help-center/help-center.component';
```

Comment:
Za ovakvu grupnu implementaciju bi trebalo da koristimo lazy loading: https://angular.io/guide/lazy-loading-ngmodules
Ovakav način utiče na performanse. Generalno, sve veće sekcije koje imaju sopstvene module, moraju da se učitavaju lazy loading metodom. Settings komponente se učitavaju recimo na ovaj način. Može da se pogleda u dashboard-routing.